.babelrc

```diff
@@ -1,5 +1,6 @@
 {
   "presets": [
+    "env",
```

Comment:
Just noticed this small change :) It shouldn't be necessary (and might actually cause problems with the module transpilation?) https://github.com/zeit/next.js/blob/v3-beta/server/build/babel/preset.js#L16